Samples/kmp-app/androidApp/src/main/java/sample/kpm_app/android/MainActivity.kt

```diff
@@ -0,0 +1,32 @@
+package sample.kpm_app.android
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import sample.kpm_app.Platform
+import sample.kpm_app.Proxy
+import java.lang.Exception
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val captureMessageBtn: Button = findViewById(R.id.captureMessageBtn)
+        val captureExceptionBtn: Button = findViewById(R.id.captureExceptionBtn)
+        val sentryProxy = Proxy()
```

Comment:
Instead of creating a `Proxy` class, we should be able to simply call a static method, such as `Sentry.captureMessage(x)`